packages/bcer-data-portal/app/src/util/bcRouteLink.util.ts

```diff
@@ -0,0 +1,111 @@
+import {
+  BCGeocoderAutocompleteData,
+  BusinessLocation,
+  RouteOptions,
+} from '@/constants/localInterfaces';
+
+export class BcRouteLinkBuilder {
+  readonly baseLink = 'https://router.api.gov.bc.ca';
+  readonly directionBaseRoute = '/directions.json?';
+  readonly optimalDirectionBaseRoute = '/optimalDirections.json?';
+
+  optimalRoute = false;
+  params: string = 'apikey=';
```

Comment:
Ahh the API key :/ I'm wondering if we should maybe proxy these calls through our API and add the API key to the request there, as to not expose the API key publicly. Unless there's a way of restricting the API Key by domain that is